nx-X11/include/Xos_r.h

```diff
@@ -476,15 +456,9 @@ typedef struct {
     (sp).sptr )
 
 #elif defined(XUSE_NETDB_R_API)
-/*
- * POSIX does not specify _r equivalents for <netdb.h> API, but some
- * vendors provide them anyway.  Use them only when explicitly asked.
- */
 # ifdef _POSIX_REENTRANT_FUNCTIONS
 #  ifndef _POSIX_THREAD_SAFE_FUNCTIONS
-#   if defined(AIXV3) || defined(AIXV4) || defined(__osf__)
-#    define X_POSIX_THREAD_SAFE_FUNCTIONS 1
-#   endif
+#   define X_POSIX_THREAD_SAFE_FUNCTIONS 1
```

Comment:
Hum, no, I think we could drop the whole section (from `# ifdef _POSIX_REENTRANT_FUNCTIONS` to `# endif`), because the offenders `AIXV3`, `AIXV4` and `__osf__` for which the macro `X_POSIX_THREAD_SAFE_FUNCTIONS` should have been set *only* are gone now.